jobs/client_count_daily_view.sh

```diff
@@ -21,10 +21,14 @@ base+="os_version"
 select+="devtools_toolbox_opened_count > 0 as devtools_toolbox_opened,"
 select+="case when distribution_id in ('canonical', 'MozillaOnline', 'yandex') "
 select+="then distribution_id else null end as top_distribution_id,"
+select+="bucketed(scalar_parent_browser_engagement_total_uri_count, "
```

Comment:
Can you indent or make the string multiline in some way to make it obvious this is a single field, and not three?

Reply:
changed the placement of spaces from trailing to leading, so that it kind-of indents.